draft-ietf-scitt-architecture.md

```diff
@@ -312,12 +298,22 @@ A Receipt is an offline, universally-verifiable proof that an entry is recorded
 Receipts do not expire, but it is possible to append new entries (more recent Signed Statements) that subsume older entries (less recent Signed Statements).
 
 Anyone with access to the Transparency Service can independently verify its consistency and review the complete list of Transparent Statements registered by each Issuer.
-However, the Registrations of separate Transparency Services are generally disjoint, though it is possible to take a Transparent Statement from one Transparency Service and register it again on another (if its policy allows), so the authorization of the Issuer and of the Transparency Service by the Verifier of the Receipt are generally independent.
+However, the Registrations on separate Transparency Services are generally disjoint, though it is possible to take a Transparent Statement, a Signed Statement with a Receipt from the first Transparency Service in its unprotected header, and register it on another Transparency Service, where the second receipt will be over the first Receipt in the unprotected header.
```

Comment:
This is new, and unclear to me. I guess the receipt should be on the on transparent statement registered at the first service (including its first receipt as untrusted header) not just on the first receipt. As discussed above, this complicates the interpretation of receipts by verifiers and auditors.

Reply:
Editors meeting note:
Transparent statements are a result of the "client" stapling the receipt to their local copy of the signed statement.
A client can resubmit the signed statement, with the receipt in the unprotected header.